CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.1.0
```

Comment:
No need to increase version as we change tests only. Please keep the previous version